microsoft/testsuites/kvm/kvm_unit_tests.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from microsoft.testsuites.kvm.kvm_unit_tests_tool import KvmUnitTests
+
+
+@TestSuiteMetadata(
+    area="kvm",
+    category="community",
+    description="""
+    This test suite for executing the community maintained KVM tests at:
+        https://gitlab.com/kvm-unit-tests/kvm-unit-tests
+    """,
+)
+class KvmUnitTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Runs the kvm-unit-tests suite for Azure VMs.
+        """,
+        priority=3,
+    )
+    def kvm_unit_tests_for_azure_vm(
```

Comment:
The test case should be able to run on any place, and use the standard name as other tests: `verify_kvm_unit_tests`

Reply:
The `expected_failures` list is actually "tuned" for Azure VMs. There might be other failures (or no failures at all) in other environments. That's why I named it this way.

Reply:
By default, test cases are for Azure, because they are in microsoft folder. But it may be used in other platforms as well. When we need that, we can just make the expected failures configurable. The test case name is a breaking change to pipelines, we should make it right.